lib/SILGen/SILGenPoly.cpp

```diff
@@ -3275,9 +3275,10 @@ static ManagedValue createAutoDiffThunk(SILGenFunction &SGF,
                                                 outputSubstTypeNotDiff);
   // `autodiff_function_extract` is consuming; copy `fn` before passing as
```

Comment:
I wonder if this comment is still accurate?

Reply:
Oops, they are outdated. I’ll handle them in a future PR.